packages/components-storybook-v2/src/stories/components/form/input.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Input } from '@trezor/components-v2';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, select } from '@storybook/addon-knobs';
+import { infoOptions } from '../../../support/info';
+
+storiesOf('Form', module).add(
+    'Input',
+    () => {
+        const value: any = text('Value', '');
+        const variant: any = select(
+            'Variant',
+            {
+                'Default (large)': null,
+                Small: 'small',
+            },
+            null
+        );
+        const display: any = select(
+            'Size',
```

Comment:
name of the knob should be the same as name of the prop (knob `size` sets prop `display`)